### DISCUSSION TITLE
Add a "sponsor" flair to usernames on Github Issues and Pull Requests

### DISCUSSION THREAD

**Discussion by zachriggle** [community/community#8778] [Other Feature Feedback, Questions, & Ideas]
While I don't expect any special treatment for being a sponsor -- I'm a sponsor because I want to support the work, not to get priority -- it would be neat if we could show some kind of flair next to the username on PRs and Issues.

I think that increasing the visibility of sponsors on a project will encourage others to do so.  In the below images, you see the team affiliation of a Github member with a project.  I think something similar for Sponsor would be neat.  

Maybe add a ⭐️ if you're feeling snazzy.

<img width="520" alt="Screen Shot 2021-12-13 at 1 07 32 AM" src="https://user-images.githubusercontent.com/111640/145767499-b0af42fd-d8ee-4076-9aa5-1569612f4097.png">

<img width="531" alt="Screen Shot 2021-12-13 at 1 05 27 AM" src="https://user-images.githubusercontent.com/111640/145767254-acfec3fd-c343-491a-8bd5-ff8949cfad4e.png">

<img width="523" alt="Screen Shot 2021-12-13 at 1 05 48 AM" src="https://user-images.githubusercontent.com/111640/145767301-2ddadaeb-fc34-4d01-aad0-0217e5876aae.png">

Additionally, I think it would be useful if the Sponsor bubble / label / etc were clickable (or hover-and-expand) to give a link to viewers to make it easy for them to become a sponsor.

**Comment by codecook-start** [comment]
Thank you.
